Formula/xctesthtmlreport.rb

```diff
@@ -0,0 +1,19 @@
+class Xctesthtmlreport < Formula
+  desc "Xcode-like HTML report for Unit and UI Tests"
+  homepage "https://github.com/XCTestHTMLReport/XCTestHTMLReport"
+  url "https://github.com/XCTestHTMLReport/XCTestHTMLReport/archive/refs/tags/2.2.2.tar.gz"
+  sha256 "1d7159e4e66feae6194bec2303690e91db17e2593b597ac0db357942020bf4a9"
+  license "MIT"
+  head "https://github.com/XCTestHTMLReport/XCTestHTMLReport.git", branch: "main"
+
+  depends_on xcode: "13.0"
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/xchtmlreport"
+  end
+
+  test do
+    assert_match "XCTestHTMLReport #{version}", shell_output("#{bin}/xchtmlreport -h | head -n 1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

-----

You also don't need to pipe to `head` when doing `assert_match`.

Reply:
Thank you! I added a more comprehensive functional test (and removed the unnecessary pipe).